### PR TITLE
Update to latest centos6 container

### PR DESCRIPTION
**Pull request by AdamBrousseau**
Version 16
Includes GH and GHE in known_hosts file

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
jenkins compile xlinux jdk11

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -407,8 +407,8 @@ x86-64_linux:
       8: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
       11: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
     docker_image:
-      8: 'eclipseopenj9/jenkins-agent-x86-centos6:15'
-      11: 'eclipseopenj9/jenkins-agent-x86-centos6:15'
+      8: 'eclipseopenj9/jenkins-agent-x86-centos6:16'
+      11: 'eclipseopenj9/jenkins-agent-x86-centos6:16'
```
Comment (keithc-ca) [inline buildenv/jenkins/variables/defaults.yml:411]: Why do we not want to use `latest` instead of `16`?

Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml:411]: This is our service mindset coming though. If we need to rebuild something old, the release branch will contain the right version of the container that was used at the time.

Comment (pshipton) [reply buildenv/jenkins/variables/defaults.yml:411]: Just a guess, but using `latest` means we could be broken unexpectedly.

**Review by pshipton — APPROVED** [review]

File: buildenv/jenkins/variables/defaults.yml (hunk 1 above)
Comment (keithc-ca) [reply buildenv/jenkins/variables/defaults.yml:411]: Got it. Thanks.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**